src/mod/applications/mod_callcenter/mod_callcenter.c

```diff
@@ -1678,6 +1759,68 @@ static switch_status_t playback_array(switch_core_session_t *session, const char
 
 	return status;
 }
+static void cc_agent_noanswer_count_status_update(const char *agent_name, const char *member_uuid, const char *member_session_uuid, switch_call_cause_t hangupcause, cc_agent_status_t newstatus, int count, int max)
```

Comment:
maybe change the noanswer to failed ?

Reply:
yes, agree better to "read", changing it.